jsonnet/kube-prometheus.libsonnet

```diff
@@ -0,0 +1,20 @@
+local k = import 'ksonnet/ksonnet.beta.3/k.libsonnet';
+local configMapList = k.core.v1.configMapList;
+
+(import 'prometheus.libsonnet') +
+(import 'ceph-mixins/mixin.libsonnet') + {
```

Comment:
How this import gets resolved?

Reply:
we pull all the dependencies to local (i.e jsonnet/vendor)